Sateltiles/admin_user.py

```diff
@@ -0,0 +1,147 @@
+
+        getLink_button = wx.Button(panel, label='GET TMS LINK', pos=(470,190+130), size=(100,40)) 
+        getLink_button.Bind(wx.EVT_BUTTON, self.getTMS_Link)
+
+        back_button = wx.Button(panel, label='BACK', pos=(10,310+130), size=(100,40)) #back home buttons
+        back_button.Bind(wx.EVT_BUTTON, self.backHome)
+
+        source_button = wx.Button(panel, label='Browse', pos=(280,227+130), size=(90,25)) 
+        source_button.Bind(wx.EVT_BUTTON, self.browseSourceFile)
+
+        save_button = wx.Button(panel, label='Browse', pos=(280,257+130), size=(90,25)) 
+        save_button.Bind(wx.EVT_BUTTON, self.browseSaveFile)
+
+        generate_button = wx.Button(panel, label='GENERATE', pos=(175,285+130), size=(90,30)) 
+        generate_button.Bind(wx.EVT_BUTTON, self.generateTMS)
+
+        id=wx.NewId()
+        self.list = wx.ListCtrl(panel,pos=(20,0+50), size=(550,260), style=wx.LC_REPORT 
+                                 | wx.BORDER_SUNKEN
+                                 #| wx.BORDER_NONE         <<< This is up to your style uncomment # to enable
+                                 #| wx.LC_EDIT_LABELS
+                                 #| wx.LC_SORT_ASCENDING
+                                 #| wx.LC_NO_HEADER
+                                 #| wx.LC_VRULES
+                                 #| wx.LC_HRULES
+                                 #| wx.LC_SINGLE_SEL
+                                  )
+        
+
+        self.list.InsertColumn(0,"Min lat")
+        self.list.InsertColumn(1,"Max lat")
+        self.list.InsertColumn(2,"Min long")
+        self.list.InsertColumn(3,"Max long")
+        self.list.InsertColumn(4,"EPSG")
+        self.list.InsertColumn(5,"Create by")
+        self.list.InsertColumn(6,"Create on")
+        self.list.InsertColumn(7,"Hide")
+
+
+        for i in range(100):
+            pos = self.list.InsertStringItem(i,"80") # 0 will insert at the start of the list
+            self.list.SetStringItem(pos,1,"26.45") # add values in the other columns on the same row
+            self.list.SetStringItem(pos,2,"88.183333")
+            self.list.SetStringItem(pos,3,"30.45")
+            self.list.SetStringItem(pos,4,"900913")
+            self.list.SetStringItem(pos,5,"Jimmy")
+            self.list.SetStringItem(pos,6,"28-06-2016")
+            self.list.SetStringItem(pos,7,"0")
+
+        self.tif_path = ""
+        self.save_path = ""
+    
+        self.list.Show(True)
+        self.Show(True)
+
+    def backHome(self, evt):
+        self.Destroy() # close LoginPage
+        log.LoginPage(None, title='LoginPage')  #open up login page
+
+    def getTMS_Link(self, evt):
+        count = self.list.GetItemCount()
+        item_data = []
+        for row in range(count):
+            if self.list.IsSelected(row): # get selected item data
+                print ("You have selected item " + str(row+1))

```

Comment:
remove these print statement...if you want other to debug the script, put some global variable at the top of file like DEBUG = False as default and user can change it to True and make some function to print in the console